contrib/win32/win32compat/misc.c

```diff
@@ -1413,3 +1457,22 @@ get_others_file_permissions(wchar_t * file_name, int isReadOnlyFile)
 		LocalFree(pSD);
 	return ret;
 }
+
+char*
+get_ssh_dir_path()
```

Comment:
consider better name of this function. ssh_dir seems like where ssh binary presents while this directory is where we store sshd resources and logs